js/its_number_prime.test.js

```diff
@@ -0,0 +1,26 @@
+const isPrime = require('./its_number_prime');
+
+
+describe('its_number_prime', () => {
+
+  test('If get 0 should return false', () => {
```

Comment:
Looks good :smile: Although, I would name it the more common way that tests are written in:

```suggestion
  test('should return false for 0', () => {
```